src/components/Navbar/ResetDrop.tsx

```diff
@@ -6,9 +6,9 @@ import { ThemeProvider } from '@emotion/react';
 import ResetGame from './ResetGame';
 import { useSelector } from 'react-redux';
 import { RootState } from '../../store';
-
+import MakeSpectator from '../RoomView/MakeSpectator';
```

Comment:
Not sure where to put this - we should rename this entire component from ResetDrop.tsx to GameSettings.tsx